src/libs/actions/Report.js

```diff
@@ -2183,6 +2196,21 @@ function clearPrivateNotesError(reportID, accountID) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {privateNotes: {[accountID]: {errors: null}}});
 }
 
+export default function getDraftPrivateNote(reportID) {
```

Comment:
I missed that one before, why this function is `export default`? Can we get rid of it? Other functions don't need it @DylanDylann

Reply:
Updated. Please help check

Reply:
Great thanks!